src/Storages/MergeTree/MergeTreeIndexInverted.cpp

```diff
@@ -735,8 +735,8 @@ MergeTreeIndexPtr invertedIndexCreator(
     const IndexDescription & index)
 {
     size_t n = index.arguments.empty() ? 0 : index.arguments[0].get<size_t>();
-    Float64 density = index.arguments.size() < 2 ? 1.0 : index.arguments[1].get<Float64>();
-    GinFilterParameters params(n, density);
+    UInt64 max_rows = index.arguments.size() < 2 ? DEFAULT_ROWS_IN_POSTINGS_LIST : index.arguments[1].get<UInt64>();
```

Comment:
`max_rows` is too unspecific. Could you please rename all mentions of it to something like `max_rows_in_posting_list` or `max_rows_per_postings`?